src/index.ts

```diff
@@ -49,7 +49,7 @@ export type {RadioProps} from './Radio'
 export {ActionList} from './ActionList'
 export type {
   ActionListProps,
-  ActionListGroupProps,
+  ActionListHeadingProps,
```

Comment:
~We should probably export both for backward compatibility~

Checked dotcom, there is no usage of these types, safe to delete